test/e2e/api/matchMedia/map-bounding-box.html

```diff
@@ -0,0 +1,101 @@
+
+                // Format the media query string to detect overlap:
+                // (xminm < xmaxq) and (xmaxm > xminq) and (yminm < ymaxq) and (ymaxm > yminq)
+                const query = `(map-projection: OSMTILE) and (map-zoom < 14) and (map-top-left-easting < ${bottomRightEasting}) and (map-bottom-right-easting > ${topLeftEasting}) and (map-bottom-right-northing < ${topLeftNorthing}) and (map-top-left-northing > ${bottomRightNorthing})`;
+
+                let matcher = map.matchMedia(query);
+                const logResults = (e) => {
+                    if (e.target.matches) {
+                        layer.checked = true;
+                        layer.removeAttribute('hidden');
+                    } else {
+                        layer.checked = false;
+                        layer.hidden = true;
+                    }
+                };
+                matcher.addEventListener('change', logResults);
+
+                // create a layer to visually represent the query as the map moves
+                let f = `<map-layer checked label="test media query"><map-meta name="projection" content="OSMTILE"></map-meta>
+                    <map-meta name="cs" content="pcrs"></map-meta><map-feature><map-properties>${query}</map-properties>
+                    <map-geometry><map-polygon><map-coordinates>${topLeftEasting} ${topLeftNorthing}
+                    ${bottomRightEasting} ${topLeftNorthing} ${bottomRightEasting} ${bottomRightNorthing} ${topLeftEasting} ${bottomRightNorthing}
+                    ${topLeftEasting} ${topLeftNorthing}</map-coordinates</map-polygon></map-geometry></map-feature></map-layer>`;
+
+                const parser = new DOMParser();
+                const layer = parser
+                    .parseFromString(f, 'text/html')
+                    .querySelector('map-layer');
+                map.appendChild(layer);
+                return { matcher, logResults };
+            }
+            document.querySelector('.getBoundingBox').addEventListener('click', getBoundingBox);
+        });
+    </script>
+
+    <mapml-viewer projection="OSMTILE" zoom="14" lat="45.406314" lon="-75.6883335" controls=""
+        controlslist="geolocation">
+        <map-layer data-testid="osm-layer" label="OpenStreetMap" checked="">
+            <map-link rel="license" title="Â© OpenStreetMap contributors CC BY-SA"
+                href="https://www.openstreetmap.org/copyright"></map-link>
+            <map-extent units="OSMTILE" checked="checked">
+                <map-input name="z" type="zoom" value="18" min="0" max="18"></map-input>
+                <map-input name="x" type="location" units="tilematrix" axis="column" min="0" max="262144"></map-input>
+                <map-input name="y" type="location" units="tilematrix" axis="row" min="0" max="262144"></map-input>
+                <map-link rel="tile" tref="https://tile.openstreetmap.org/{z}/{x}/{y}.png"></map-link>
```

Comment:
I have been starting to create tests that have no tiles in them at all, so that there's no network waiting going on, or minimal if possible.

Reply:
By that, does it mean we just won't have a base map for test cases?

Reply:
Here's an example: https://github.com/Maps4HTML/MapML.js/blob/main/test/e2e/elements/map-layer/layer-extent.html